app/src/main/java/me/ccrama/redditslide/Activities/AlbumPager.java

```diff
@@ -758,125 +749,6 @@ public void onClick(DialogInterface dialog, int which) {
 
     }
 
-    public void showNotifPhoto(final File localAbsoluteFilePath, final Bitmap loadedImage) {
```

Comment:
I do not really know what this was doing. Removing it seemed to make no change. If you could explain its purpose I can try to test if I broke everything.